src/Visualization/Python/GenerateXdmf.py

```diff
@@ -6,7 +6,7 @@
 import glob
 import h5py
 import numpy as np
-
```

Comment:
Keep the second blank line after the includes please (it’s good to run autopep8 over your python code)